v2/tools/generator/internal/astmodel/kubernetes_admissions_validator.go

```diff
@@ -65,14 +65,16 @@ func (v *ValidatorBuilder) AddValidation(kind ValidationKind, f *resourceFunctio
 // as well as helper functions that allow additional handcrafted validations to be injected by
 // implementing the genruntime.Validator interface.
 func (v *ValidatorBuilder) ToInterfaceImplementation() *InterfaceImplementation {
-	lpr, ok := v.resourceName.PackageReference.AsLocalPackage()
+	group, version, ok := v.resourceName.PackageReference.GroupVersion()
 	if !ok {
-		panic(fmt.Sprintf("expected resource name %s to be a local package reference", v.resourceName.String()))
+		panic(fmt.Sprintf("expected resource name %s to not have an external package reference", v.resourceName))
```

Comment:
Wondering if we should rephrase this to:
`unexpected external package reference for resource name %s`?

Reply:
Done.